templates/vpc/content/vpc.tf

```
@@ -0,0 +1,9 @@
+#
+# VPC
+#
+
+module "vpc" {
+  source            = "github.com/cds-snc/terraform-modules//vpc?ref=v9.2.5"
+  name              = ${{ values.vpc_name | dump }}
+  billing_tag_value = ${{ values.billing_tag_value| dump }} 
+}
```

One more config option for Cloud Based Sensor.

1. Is this a staging or production service?  If yes, please provide your AWS target account ID.

```suggestion
  enable_flow_log = false
}

resource "aws_flow_log" "cloud_based_sensor" {
  log_destination      = "arn:aws:s3:::cbs-satellite-${{ values.account_id | dump }}/vpc_flow_logs/"
  log_destination_type = "s3"
  traffic_type         = "ALL"
  vpc_id               = module.vpc.vpc_id
  log_format           = "$${vpc-id} $${version} $${account-id} $${interface-id} $${srcaddr} $${dstaddr} $${srcport} $${dstport} $${protocol} $${packets} $${bytes} $${start} $${end} $${action} $${log-status} $${subnet-id} $${instance-id}"

  tags = {
    CostCentre = ${{ values.billing_tag_value | dump }}
    Terraform  = true
  }
}
```

---

I have added a counter to the cloud based sensor to create it if it is a staging or production account. 

Thanks for all the suggestions and feedback Pat!